Polished "Transfering Tokens" intro (#1109)
casper-network/docs: source/docs/casper/developers/cli/transfers/index.md

# Introduction

> The following topics describe how to use the Casper command-line client to transfer tokens between purses on a Casper network. Depending on the account configuration, either a direct transfer or a multiple-signature (multi-sig) deploy transfer can be utilized.

Reviewer: ```suggestion
The following topics describe using the Casper command-line client to transfer tokens between purses on a Casper network. Depending on the account configuration, a direct transfer or a multiple-signature (multi-sig) deploy transfer can be utilized.
```

> **Transferring Tokens Using Direct Transfer**

Reviewer: Thanks for catching the discrepancy! However, since this is not a title, we should probably not capitalize anything.
```suggestion
**Transferring tokens using a direct transfer**
```

Tokens can be transferred directly when the signing key has enough weight to approve the transaction. This is the most common scenario, applicable by default for accounts with a single primary key. To use a direct transfer, see [Transferring Tokens Using Direct Transfer](./direct-token-transfer.md).

> **Transferring Tokens Using Multi-sig Deploy**

Reviewer: ```suggestion
**Transferring tokens using a multi-sig deploy**
```
